contracts/script/releases/Env.sol

```diff
@@ -0,0 +1,373 @@
+        return EigenDADisperserRegistry(_deployedProxy("DisperserRegistry"));
+    }
+
+    /// @dev Usage: `Env.impl.disperserRegistry()`
+    function disperserRegistry(DeployedImpl) internal view returns (EigenDADisperserRegistry) {
+        return EigenDADisperserRegistry(_deployedImpl("DisperserRegistry"));
+    }
+
+    /// @dev Usage: `Env.proxy.paymentVault()`
+    function paymentVault(DeployedProxy) internal view returns (PaymentVault) {
+        return PaymentVault(payable(_deployedProxy(type(PaymentVault).name)));
+    }
+
+    /// @dev Usage: `Env.impl.paymentVault()`
+    function paymentVault(DeployedImpl) internal view returns (PaymentVault) {
+        return PaymentVault(payable(_deployedImpl(type(PaymentVault).name)));
+    }
+
+    /// @dev Usage: `Env.impl.accessControl()`
+    function accessControl(DeployedImpl) internal view returns (EigenDAAccessControl) {
+        return EigenDAAccessControl(_deployedImpl("AccessControl"));
+    }
+
+    /// @dev Usage: `Env.impl.operatorStateRetriever()`
+    function operatorStateRetriever(DeployedImpl) internal view returns (OperatorStateRetriever) {
+        return OperatorStateRetriever(_deployedImpl(type(OperatorStateRetriever).name));
+    }
+
+    /// @dev Usage: `Env.impl.pauserRegistry()`
+    function pauserRegistry(DeployedImpl) internal view returns (IPauserRegistry) {
+        return IPauserRegistry(_deployedImpl("PauserRegistry"));
+    }
+
+    /// -----------------------------------------------------------------------
+    /// Periphery Contracts
+    /// -----------------------------------------------------------------------
+
+    /// @dev Usage: `Env.proxy.ejectionManager()`
+    function ejectionManager(DeployedProxy) internal view returns (EigenDAEjectionManager) {
+        return EigenDAEjectionManager(_deployedProxy("EjectionManager"));
+    }
+
+    /// @dev Usage: `Env.impl.ejectionManager()`
+    function ejectionManager(DeployedImpl) internal view returns (EigenDAEjectionManager) {
+        return EigenDAEjectionManager(_deployedImpl("EjectionManager"));
+    }
+
+    /// -----------------------------------------------------------------------
+    /// Cert Verification Contracts
+    /// -----------------------------------------------------------------------
+
+    /// @dev Usage: `Env.impl.certVerifier()`
+    function certVerifier(DeployedImpl) internal view returns (EigenDACertVerifier) {
+        return EigenDACertVerifier(_deployedImpl("CertVerifier"));
+    }
+
+    /// @dev Usage: `Env.proxy.certVerifierRouter()`
+    function certVerifierRouter(DeployedProxy) internal view returns (EigenDACertVerifierRouter) {
+        return EigenDACertVerifierRouter(_deployedProxy("CertVerifierRouter"));
+    }
+
+    /// @dev Usage: `Env.impl.certVerifierRouter()`
+    function certVerifierRouter(DeployedImpl) internal view returns (EigenDACertVerifierRouter) {
+        return EigenDACertVerifierRouter(_deployedImpl("CertVerifierRouter"));
+    }
```

Comment:
imo we should only care about the router and not the immutable verifier for this system wide updates

cc @samlaf